sys/pic32/clang-config.mk

```diff
@@ -22,6 +22,9 @@ ifeq ($(LLVMBIN),)
     LLVMBIN = $(dir $(wildcard /usr/local/Cellar/llvm@18/18.*/bin/clang-18))
 endif
 
+ifeq ($(LLVMBIN),)
+    LLVMBIN = $(dir $(wildcard /usr/bin/clang))
```

Comment:
This fails on MacOS, as native clang does not support target `mipsel`. I should have it removed from the top level `target.mk` as well.